### ISSUE TITLE
Way to disable logging unless an error occurs?

### ISSUE THREAD

**Issue by owens-ben** [tzurp/cleanup-total#1]
Not a major issue, but the logging is a bit unnecessary. I don't really feel like implementing a custom reporter, it would be nice to just have a debug flag or something to reduce logging.

**Comment by tzurp** [comment]
@owens-ben thanks for your remark. Indeed it's a bug (: I fixed it in the latest update (v1.0.2) which enables logging suppression using the _customLoggerMethod_ option with an empty function:

```
services: [
        ['cleanuptotal',
            {
                customLoggerMethod: ()=> {}
            }]
    ]
```

**Edit:** I'm aware that the bug fix is not the full answer to your request. I'll check it too.

**Comment by tzurp** [comment]
@owens-ben I added a service option to log only upon errors (from v1.0.3). Configure as follows:

```
services: [
        ['cleanuptotal',
            {
                logErrorsOnly: true
            }]
    ]
```
